mk/spksrc.service.start-stop-status

```diff
@@ -61,9 +58,9 @@ start_daemon ()
                 SU=""
             fi
             if [ -z "${SVC_BACKGROUND}" ]; then
-                $SU ${SERVICE_SHELL} -c "${SVC_CD}${SERVICE_COMMAND}" >> ${OUT} 2>&1
+                $SU /bin/sh -c "${SVC_CD}${SERVICE_COMMAND}" >> ${OUT} 2>&1
```

Comment:
The idea here is to be able to use shell specific (dash, bash...) instructions in "SERVICE_COMMAND". But I agree that this feature has never been used right now.

Reply:
I first wanted to rename `SERVICE_SHELL` to `SVC_SHELL` for consistency.
But as this is not used anywhere, I decided to remove it.